src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -154,38 +156,31 @@ public void actionPerformed(ActionEvent e) {
 
         EasyBind.subscribe(searchField.textProperty(), searchText -> performSearch());
 
-        /*
-        String endSearch = "endSearch";
-        searchField.getInputMap().put(Globals.getKeyPrefs().getKey(KeyBinding.CLEAR_SEARCH), endSearch);
```

Comment:
Do these keyboard shortcuts now work as expected?
Would be nice if they do otherwise I would prefer to keep the commented-out code.